Test/Test Utilities/TestUtilities.swift

```diff
@@ -279,14 +279,21 @@ class AblyTests {
         }
     }
 
-    class func waitFor<T>(timeout: DispatchTimeInterval, file: FileString = #file, line: UInt = #line, f: @escaping (@escaping (T?) -> Void) -> Void) -> T? {
+    enum WaitError: Error {
```

Comment:
I don't think this is a good name for enum, because there are no wait errors other than a timeout. Maybe to introduce a common for the whole test suit enum and collect all error cases there?

Reply:
Thanks @maratal — I've followed your suggestion and changed the type to `AblyTests.Error`.